lib/python/qmk/cli/new/keymap.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+"""
+This script automates the copying of the default keymap into your own keymap.
+"""
+import os
+import sys
+
+# TODO (orngshdw): need to figure out where this script will be called from to prevent
+#   the "ModuleNotFoundError: No module named 'milc'" error
+#   am executing file from the lib/python/
+path = os.path.abspath(".")
+sys.path.append(path)
```

Comment:
I wasn't sure how we'll be calling this script. Been calling it directly and added these `os.path.abspath` & `sys` commands so that I can test the script. Will need guidance here.

Reply:
I don't have time to look at everything but I did notice this- you can remove those lines and the shebang and then call this with `bin/qmk new-keymap`. That's the preferred way to run these.

Reply:
Removed lines so am resolving.